pyartcd/tests/pipelines/test_update_golang.py

```
@@ -452,6 +452,143 @@ def test_get_content_repo_url_suffix(self, mock_konflux_db):
         url = pipeline.get_content_repo_url_suffix(8)
         self.assertEqual(url, "/brewroot/repos/rhaos-4.16-rhel-8-build/latest")
 
+    @patch("pyartcd.pipelines.update_golang.KonfluxDb")
+    def test_get_builder_pullspec(self, mock_konflux_db):
+        """Test stream-update pullspec uses the published registry.redhat.io form"""
+        mock_runtime = Mock(
+            dry_run=False,
+            working_dir=Path("/tmp/working"),
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/working"

(S108)